extension-manifest-v3/package.json

```diff
@@ -4,11 +4,12 @@
   "version": "10.2.15",
   "type": "module",
   "scripts": {
-    "postinstall": "npm run download-engines && npm run download-whotracksme-bloomfilter",
-    "build": "node scripts/build.js",
-    "start": "npm run build -- --watch",
     "download-engines": "node scripts/download-engines/index.js",
     "download-whotracksme-bloomfilter": "node scripts/download-whotracksme-bloomfilter.js",
+    "download": "npm run download-engines && npm run download-whotracksme-bloomfilter",
```

Comment:
I am fine with moving downloading into the build step. As we will have it there, I would strongly advice to refactor those scripts, so they will be "run" inside of the JS, rather than commands - they are no longer needed here. 

As you added `staging` argument to build command, we can also add `--skip-download`, or even flip it over, and add `--download` - so we run this excplicitly on CI, or `release` command, but not all the time.

Reply:
I would keep them as independent scripts for sake of local development